DataEngineering/sc.py

```diff
@@ -0,0 +1,36 @@
+import json, requests
+import sqlite3
+
+
+conn = sqlite3.connect('Bike.db')
+c = conn.cursor()
+c.execute('CREATE TABLE if not exists realtimewithT (Id,value INTEGER, timestamp)')

```

Comment:
How would the user make a query like 'what was the availability of bikes at bikestand X at 5PM yesterday?'
Try to capture as much data as possible from the API request here - all data is valuable.